1-mandatory/2-people-i-know.js

```diff
@@ -378,14 +378,22 @@ First, I want you to find all of my friends who are 35 or older.
 */
 
 let thirtyFiveOrOlder = [];
+thirtyFiveOrOlder = friends.filter((friend) => {
+  return friend.age >= 35;
+});
 
 /*
 3) Find the email address
 Next, I want you to find all of my friends who work for "POWERNET" and then store their emails in the array below
 */
 
 let powerNetEmails = [];
-
+let powerNetFriends = friends.filter((friend) => {
+  return friend.company === "POWERNET";
+});
+powerNetEmails = powerNetFriends.map((friend) => {
```

Comment:
Praise: good use of `map` here
Nitpick: As above this could be simpler and more readable as
```
powerNetFriends.map(friend => friend.email;);
```